src/scripts/main.js

```diff
@@ -1,3 +1,51 @@
 'use strict';
 
 // write code here
+const list = document.querySelectorAll('li');
+
+function getEmployees(employeesList) {
+  const employees = [];
+
+  employeesList.forEach((employe) => {
+    employees.push({
+      name: employe.textContent.trim(),
+      position: employe.getAttribute('data-position'),
+      salary: employe.getAttribute('data-salary'),
+      age: employe.getAttribute('data-age'),
+    });
+  });
+
+  return employees;
+}
+
+getEmployees(list);
+
+function sortList(employeesList) {
+  const listCopy = [...employeesList];
+
+  const salaryNum = (salary) => salary.slice(1).split(',').join('');
+
+  return listCopy.sort((employeA, employeB) => {
+    const salaryA = salaryNum(employeA.getAttribute('data-salary'));
+    const salaryB = salaryNum(employeB.getAttribute('data-salary'));
```

Comment:
The `getAttribute` method is being used on objects in `employeesList`, which are not DOM elements. You should access the `salary` property directly from the employee objects.